src/main/java/uk/gov/hmcts/ccd/data/definition/CachedCaseDefinitionRepository.java

```diff
@@ -49,12 +56,19 @@ public CachedCaseDefinitionRepository(@Qualifier(DefaultCaseDefinitionRepository
     @Override
     public CaseTypeDefinition getCaseType(final String caseTypeId) {
         CaseTypeDefinitionVersion latestVersion = this.getLatestVersion(caseTypeId);
-        return caseDefinitionRepository.getCaseType(latestVersion.getVersion(), caseTypeId);
+        return this.getCaseType(latestVersion.getVersion(), caseTypeId);
     }
 
     @Override
-    public CaseTypeDefinition getCaseType(int version, String caseTypeId) {
-        return caseDefinitionRepository.getCaseType(version, caseTypeId);
+    public CaseTypeDefinition getCaseType(final int version, final String caseTypeId) {
+        boolean withinTimeInterval = LocalDateTime.now().getHour() >= 1 && LocalDateTime.now().getHour() <= 5;
```

Comment:
Maybe introduce properties for `definition.cache.request-scope.start-hour` and `definition.cache.request-scope.end-hour`, for flexibility

Reply:
Actually removed time interval check